telescope/build.gradle

```diff
@@ -16,8 +20,8 @@ android {
   }
 
   compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_8
-    targetCompatibility JavaVersion.VERSION_1_8
+    sourceCompatibility JavaVersion.VERSION_17
+    targetCompatibility JavaVersion.VERSION_17
```

Comment:
Seems high. 11?

Reply:
AGP yelled at me. In hindsight it probably just wanted me to build the project using 17 in CI and not change this value, but I wasn't thinking and bumped this with it.